app/components/troubleshooting_options_component.html.erb

```diff
@@ -1,10 +1,4 @@
 <%= tag.section(**tag_options, class: css_class) do %>
-  <% if new_features? %>
```

Comment:
Can we remove `new_features?` now? From
* https://github.com/18F/identity-idp/blob/main/app/components/troubleshooting_options_component.rb#L16
* https://github.com/18F/identity-idp/blob/main/app/views/idv/session_errors/_in_person_proofing_options.html.erb#L2
* https://github.com/18F/identity-idp/blob/main/spec/components/troubleshooting_options_component_spec.rb#L33